source/ShopPanel.cpp

```diff
@@ -366,7 +367,7 @@ bool ShopPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, boo
 	{
 		if(!CanSell(toStorage))
 			FailSell(toStorage);
-		else
+		if(CanSell(toStorage))
```

Comment:
why?

Reply:
It was needed previously, but not anymore.